src/config.ts

```diff
@@ -7,6 +7,7 @@ export default (): any => ({
   },
   database: {
     /** The URI to connect to the database */
-    uri: process.env.MONGO_URI || 'mongodb://localhost:27017/damplab'
+    uri: process.env.MONGO_URI || 'mongodb://localhost:27017/damplab',
+    debug: true
```

Comment:
What is this debug value for?

Reply:
oops sorry! added that a while ago when trying to debug things but it didn't do anything and forgot to remove it